releaseNotes/KubernetesProvider.md

```diff
@@ -0,0 +1,23 @@
+# Azure App Configuration Kubernetes Provider
+
+[Image][image] | [Sample][sample]
+
+## 1.0.0-preview - April 07, 2023
+
+### New Features
+
+* Resolving Key Vault References and download them to Kubernetes Secret

```

Comment:
Added support for Key Vault references, which allows resolved secrets to be saved to a Kubernetes Secret.